libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -46,60 +46,69 @@ class AC_PosControl
     CLASS_NO_COPY(AC_PosControl);
 
     /// set_dt / get_dt - dt is the time in seconds since the last time the position controllers were updated
-    ///   _dt should be set based on the time of the last IMU read used by these controllers
+    ///   _dt_s should be set based on the time of the last IMU read used by these controllers
     ///   the position controller should run updates for active controllers on each loop to ensure normal operation
-    void set_dt(float dt) { _dt = dt; }
-    float get_dt() const { return _dt; }
+    void set_dt_s(float dt) { _dt_s = dt; }
+    float get_dt_s() const { return _dt_s; }
 
     // Updates internal position and velocity estimates in the NED frame.
     // Falls back to vertical-only estimates if full NED data is unavailable.
     // When high_vibes is true, forces use of vertical fallback for velocity.
     void update_estimates(bool high_vibes = false);
 
-    /// get_shaping_jerk_NE_cmsss - gets the jerk limit of the ne kinematic path generation in cm/s/s/s
+    /// get_shaping_jerk_NE_msss - gets the jerk limit of the ne kinematic path generation in cm/s/s/s
     float get_shaping_jerk_NE_cmsss() const { return _shaping_jerk_ne_msss * 100.0; }
```

Comment:
Its quite marginal on all of these,  but I would vote to use the meters method. Just makes it less likely that we forget to update one or something.

```suggestion
    float get_shaping_jerk_NE_cmsss() const { return get_shaping_jerk_NE_msss() * 100.0; }
```

Reply:
I agree.